en/docs/develop/customizations/customizing-login-pages-for-dev-portal-and-publisher.md

```diff
@@ -192,6 +192,34 @@ Follow the steps below to customize them.
         ```
         <APIM-HOME>/repository/deployment/server/jaggeryapps/devportal/site/public/tenant_themes/<tenant-name>/login/css/loginTheme.css
         ```
+    !!! note "Tenant-wise theming"
+        You can have a different theme for the login pages of different tenants. To do this, do the following.
+        
+        1. Create a folder called **themes** inside the `<APIM-HOME>/repository/deployment/server/webapps/authenticationendpoint/` directory and add the custom copied files you created in this step.
```

Comment:
Should this be done to the accountrecoveryendpoint app as well?

Reply:
Yes. Works with that as well, Fixed with https://github.com/wso2/docs-apim/pull/6327/commits/e9b04e931382f5512594763aea87be8d843ae3d6